tgui/packages/tgui/interfaces/Techweb.js

```diff
@@ -721,8 +721,9 @@ const TechNode = (props, context) => {
           <Button
             key={id}
             className={`${design_cache[k].class} Techweb__DesignIcon`}
-            tooltip={design_cache[k].name}
-            tooltipPosition={i % 15 < 7 ? "right" : "left"} />
+            // tooltip={design_cache[k].name}
+            // tooltipPosition={i % 15 < 7 ? "right" : "left"}
```

Comment:
should we really be leaving comments? i'd rather not leave dead code in here like this

if maintainers don't agree with me then maybe at least explain why it's commented out so someone doesn't uncomment it in the future not knowing why it is how it is

Reply:
it's because it's a bandaid. These tooltips can then be easily re-added once the problem with tooltips performance is solved. I'll leave an extra clarification on it.